engines/router/missionctl/instrumentation/prometheus.go

```diff
@@ -32,6 +32,8 @@ var requestLatencyBuckets = []float64{
 	2000, 5000, 10000, 20000, 50000, 100000,
 }
 
+var additionalRegisteredMetricNames = set.New(nil)
```

Comment:
Could you add a comment above `additionalRegisteredMetricNames` why it's needed? Just a one-liner explaining that it's meant to prevent re-registrations if the Register method is called multiple times will be useful.